src/controller.rs

```diff
@@ -72,10 +80,12 @@ impl FoxBox {
     pub fn new(verbose: bool,
                hostname: Option<String>,
                http_port: u16,
-               ws_port: u16) -> Self {
-
+               ws_port: u16, tls_enabled: bool) -> Self {
```

Comment:
uber-nit: follow the pattern of previous params by moving `tls_enabled` to the next line